### DISCUSSION TITLE
Size of Converted CR2 Image

### DISCUSSION THREAD

**Discussion by SteveRowl** [ImageMagick/ImageMagick#6519] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

Win 10 64

### Description

I am using IM to convert Canon CR2 (600D) files to FITS format for astronomical purposes. A CR2 file of approx. 24MB converts to a FITS file of size approx. 105MB.

My question is: Why is the FITS image size so much larger than the CR2 image?

**Comment by snibgo** [comment]
CR2 files are compressed. FITS files are not compressed.

EDIT: More importantly, the raw data in the CR2 has only one channel per pixel, probably 14 bits. The FITS file has 3 channels per pixel, probably 16*3 = 48 bits.

**Comment by SteveRowl** [comment]
Thanks for the explanation.
